myhdl/_Cosimulation.py

```diff
@@ -137,7 +137,10 @@ def _get(self):
                     if next >= (1 << (s._nrbits-1)):
                         next |= (-1 << s._nrbits)
             except ValueError:
-                next = intbv(0)
+                if v in 'xX':
+                    next = s._init
```

Comment:
Suppose I have a single tri-state output buffer on a bus. Its output enable is initially off, so the bus's value will remain at 'X'. Only after OE turned on for once and then turned off, the bus's value will change to 'Z'.

Essentially we are mapping 4-value (01XZ) logic to a 3-value (01None) logic. I think Z state is more important than 'X', because it's required in tri-state signal. As for 'X' state, assigning any of '01Z' to it is not technically wrong.

In short, the mapping comes to my mind is:
```
0 -> 0
1 -> 1
Z -> None
X -> s._init    # None for tristate, 0/1 for others.
```